tool/grind.dart

```diff
@@ -6,38 +6,68 @@ import 'dart:convert';
 import 'dart:io';
 
 import 'package:grinder/grinder.dart';
+import 'package:http/http.dart' as http;
 
 import 'common.dart';
 
-main(List<String> args) => grind(args);
+Future<void> main(List<String> args) => grind(args);
 
 @DefaultTask()
-@Depends(colors, icons, catalog, version)
-generate() => null;
+@Depends(analysisOptions, colors, icons, catalog)
```

Comment:
Is removing `version` intentional?

Reply:
Yep!

(No 😞  bad conflict resolution). Fixed!